models/Post.js

```diff
@@ -0,0 +1,21 @@
+// This is a code snippet, not used for production.
+
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const postSchema = new Schema({
+  title: String,
+  postedBy: {
+      type: mongoose.Schema.Types.ObjectId,
```

Comment:
We need to decide either to stick with ObjectIds and add them to the user/player schemas or instead use something like the username here.